### PR TITLE
[SUPPORTENG-905] Update PKCE source code sample

### PR DESCRIPTION
**Pull request by iscorer**
https://zapierorg.atlassian.net/browse/SUPPORTENG-905

### CODE REVIEW

**Review by eliangcs — DISMISSED** [review]
Makes sense but I'm not 100% sure. @standielpls can you take a look too? Thanks!

File: packages/cli/snippets/oauth2-pkce.js [hunk 1]
```diff
@@ -14,6 +14,8 @@ const authentication = {
         state: '{{bundle.inputData.state}}',
         redirect_uri: '{{bundle.inputData.redirect_uri}}',
         response_type: 'code',
+        code_challenge_method: 'S256', // Added for PKCE
+        code_challenge: '{{bundle.inputData.code_challenge}}', // Added for PKCE
```
Comment (standielpls) [inline packages/cli/snippets/oauth2-pkce.js:18]: I don't think these are necessary. Our backend computes those values for us, and we don't read them from the config.

File: packages/cli/snippets/oauth2-pkce.js [hunk 2]
```diff
@@ -14,6 +14,8 @@ const authentication = {
         state: '{{bundle.inputData.state}}',
         redirect_uri: '{{bundle.inputData.redirect_uri}}',
         response_type: 'code',
+        code_challenge_method: 'S256', // Added for PKCE
```
Comment (eliangcs) [inline packages/cli/snippets/oauth2-pkce.js:17]: The backend currently always return `S256`, but it's better for the client to not assume that.

```suggestion
        code_challenge_method: '{{bundle.inputData.code_challenge_method}}', // Added for PKCE
```

**Comment by iscorer** [comment]
I seem to have got myself into a bit of a mess by trying to change the commit associated with the PR. I wanted to implement a different set of changes based on the discussion [here](https://stackoverflow.com/questions/36168839/how-to-remove-commits-from-a-pull-request).

I followed [this process](https://stackoverflow.com/questions/36168839/how-to-remove-commits-from-a-pull-request) to do that but I've ended up with a messy commit history here. My apologies.

Would it be easier for me to close this PR and create a new PR with a new branch after pulling from `main`?

**Comment by eliangcs** [comment]
@iscorer yes, feel free to create a new PR if you think that's easier.

**Comment by iscorer** [comment]
Thank you @eliangcs 👍 Closing this PR. Will create a new one.

**Pull request closed without merging**